bin/gobblin-mapreduce.sh

```diff
@@ -136,13 +137,16 @@ LIBJARS=(
   $FWDIR_LIB/guava-15.0.jar
   $FWDIR_LIB/avro-1.7.7.jar
   $FWDIR_LIB/avro-mapred-1.7.7-hadoop2.jar
-  $FWDIR_LIB/metrics-core-3.1.0.jar
-  $FWDIR_LIB/metrics-graphite-3.1.0.jar
-  $FWDIR_LIB/gson-2.3.1.jar
-  $FWDIR_LIB/joda-time-2.9.jar
-  $FWDIR_LIB/data-1.15.9.jar
   $FWDIR_LIB/commons-lang3-3.4.jar
   $FWDIR_LIB/config-1.2.1.jar
+  $FWDIR_LIB/data-1.15.9.jar
+  $FWDIR_LIB/gson-2.3.1.jar
+  $FWDIR_LIB/joda-time-2.9.jar
+  $FWDIR_LIB/kafka-clients-0.8.2.1.jar
+  $FWDIR_LIB/metrics-core-2.2.0.jar
+  $FWDIR_LIB/metrics-core-3.1.0.jar
```

Comment:
That would be ideal, yes, but I don't know the level of effort. I'll just merge this for now, and leave the removal of 2.2.0 as a TODO.